synfig-studio/plugins/lottie-exporter/plugin.xml.in

```diff
@@ -11,4 +11,14 @@
        <_description>Lottie HTML Preview (*.html)</_description>
        <exec stdout="log">lottie-exporter.py</exec>
    </exporter>
+   <exporter>
+       <extension>html</extension>
+       <_description>Lottie HTML Preview without variable widths (*.html)</_description>
+       <exec stdout="log" interpreter="python">export_without_variable_width.py</exec>
+   </exporter>
+   <exporter>
+       <extension>json</extension>
+       <_description>Lottie HTML Preview without variable widths(*.json)</_description>
```

Comment:
-> `<_description>Lottie Animation without variable widths (*.json)</_description>`

Reply:
Why does the tag name start with `_`?

Reply:
This indicates tag contents as translatable.

Reply:
@morevnaproject  changed the description content.